abdulfataiaka/pluralsight-admin
# #161348354 Implement 404 content for manage course

#### What does this PR do
- Implement 404 content for manage course

#### Description of work done
- View 404 content for invalid course id

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Select a course for edit
- Change the course ID on the url to an invalid one
- Ensure to see a 404 content

#### Relevant pivotal tracker story
[161348354](https://www.pivotaltracker.com/story/show/161348354)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 7 28 19 pm" src="https://user-images.githubusercontent.com/32123313/47236814-26f6bf80-d3d5-11e8-9ea6-cce34e2c2fb6.png">
